specutils/io/read_fits.py

```diff
@@ -209,6 +209,12 @@ def read_affine_transforms(self, wcs_dim=None):
         transform_matrix = self.read_transform_matrix(wcs_dim, cdelt=
         affine_transform_dict['cdelt'])
 
+        # DC-FLAG (IRAF standard, e.g. http://iraf.net/irafdocs/specwcs.php)
+        try:
+            affine_transform_dict['DC-FLAG'] = self.fits_header['DC-FLAG']
```

Comment:
there is an easier way of writing this:
`self.fits_header.get('DC-FLAG', 0)` - this will automatically do the try and except and return 0 if not found.